### PR TITLE
add catalog

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
- [DEMO LINK](https://DidenkoDenis306.github.io/layout_catalog/)
- [TEST REPORT LINK](https://DidenkoDenis306.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,334 @@
...
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a target="_blank" href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
```
Comment (Ihor-Karpyn) [inline src/index.html:18]: Writing meaningful alt attributes for the img tag is crucial for several reasons:
1. Accessibility: Alt attributes provide a textual description of an image, which is essential for visually impaired users who rely on screen readers. Meaningful alt text allows these users to understand the content and context of the image, ensuring equal access to information.
2. SEO benefits: Search engines like Google use alt text to index and rank images in search results. Including descriptive alt text improves your website's SEO, making it more discoverable and increasing organic traffic.
3. Usability: Meaningful alt text improves the overall user experience. If an image fails to load, the alt text will be displayed, allowing users to understand the intended message or content.
In conclusion, writing meaningful alt attributes for img tags is essential for enhancing accessibility, boosting SEO, improving usability, and ensuring compliance with legal requirements. It is a simple yet powerful way to create a more inclusive and effective online experience for all users.

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,334 @@
...
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <section class="catalog">
+        <div class="catalog__container">
+          <article class="catalog__card card" data-qa="card">
+
+            <img
```
Comment (Ihor-Karpyn) [inline src/index.html:65]: Add blank lines between child elements but don't add between parent and child

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,334 @@
...
+              src="./images/imac.jpeg"
+              class="card__product-image"
+              alt="iMac"
```
Comment (Ihor-Karpyn) [inline src/index.html:68]: alt, fix in all cases

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      margin-bottom: 4px;
+    }
+
+    &-code {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $color-secondary;
+      margin-bottom: 16px;
+    }
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/card.scss:33]: All of these classes are elements of the card, move it to level up. Fix in all cases

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/card.scss:2]: Move box-sizing to normalize css

File: src/styles/blocks/nav.scss [hunk 6]
```diff
@@ -0,0 +1,55 @@
+.nav {
+  &__list {
+    list-style: none;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/nav.scss:3]: move to normalize css

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
GJ, almost done

**Review by Ihor-Karpyn — APPROVED** [review]
🔥
